README.md

```diff
@@ -1,33 +1,99 @@
 # reNFT subgraphs
 
-To get started, first, globally install `graph` binary:
+## Getting Started
 
-`yarn global add @graphprotocol/graph-cli`
+First, globally install `graph` binary:
 
-depending on whether you are using `yarn`. We suggest you use `yarn` instead of `npm`.
+```
+yarn global add @graphprotocol/graph-cli
+```
 
-This command will install `graph` binary on your machine. If after the installation you don't have the binary, add it to your path. You can figure out where `yarn` installs global binaries by running:
+> We suggest you use `yarn` instead of `npm`.
 
-`yarn global bin`
+This command will install the `graph` binary on your machine. If you still don't have the binary after the installation, you can add it to your path. You can figure out where `yarn` installs global binaries by running:
 
-take this directory and then add to your path:
+```
+yarn global bin
+```
 
-```export PATH="$PATH:`yarn global bin`"```
+Take this yarn directory and then add to your path:
+
+```
+export PATH="$PATH:`yarn global bin`"
+```
+
+Consider familiarising yourself with [The Graph FAQ](https://thegraph.com/docs/en/developing/developer-faqs/) as well.
+
+## Installation
 
 This repository is a [yarn workspace](https://classic.yarnpkg.com/lang/en/docs/workspaces/).
 
-To install deps, in the root run `yarn install`.
+To install dependencies, run:
+
+```
+yarn install
+```
+
+To execute scripts specific to a workspace, you can run:
+
+```
+yarn workspace @renft/subgraphs-azrael codegen
+```
+
+This would execute the `codegen` script in `subgraphs/azrael`
+
+## Adding RPC urls
+
+To use the subgraphs, you must specify a space-separated list of RPCs to add to the graph node in the .env file.
+
+Each RPC will be tagged with the name of the network, followed by the RPC. Examples of the formatting for RPC urls can be found in `.env.example`. The list of supported networks can be found [here](https://thegraph.com/docs/en/developing/supported-networks/).
+
+## Notes on using Anvil and other local RPCs
+
+This docker compose file uses `host.docker.internal` which allows docker containers to access ports on the host machine. 
+
+When using localhost RPCs, such as Anvil, replace `localhost` with `host.docker.internal` before adding the RPC to the .env file. For example, `https://localhost:8545` should become `http://host.docker.internal:8545`.
+
+More info on this topic can be found [here](https://medium.com/@TimvanBaarsen/how-to-connect-to-the-docker-host-from-inside-a-docker-container-112b4c71bc66).
+
+## Running the graph node
+
+To start the local graph node, you will need `docker`, `docker-compose`, and `jq`. Then, run:
```

Comment:
let's just mention `docker` and remove `docker-compose`. Moreover, let's mention that people should use `Docker version 24.0.0, build 98fdcd7` and up